swagger.yml

```diff
@@ -138,11 +130,11 @@ paths:
         "204":
           description: "No Content"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+           description:  "Authentication information is missing or invalid"
```

Comment:
Please use the parameterised values here, there is no point repeating the messages again. The last block has the common 401, 403 and 503 parameterised